src/sentry/static/sentry/less/shared-components.less

```diff
@@ -1348,22 +1348,29 @@ table.integrations {
       margin: 0;
     }
 
-    li a {
-      padding: 5px 8px;
-      display: block;
-      padding-left: 33px;
-      position: relative;
-      max-width: 100%;
-      overflow: hidden;
-      text-overflow: ellipsis;
-      text-align: left;
+    li {
+      strong {
```

Comment:
@ckj This is copy/paste from `.project-dropdown`, so not sure if there's a way to DRY it up, but I'm open to it.